pkg/adapter/applicationLogs_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapter
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestSeverity(t *testing.T) {
+	severities := []Severity{
+		Default,
+		Debug,
+		Info,
+		Notice,
+		Warning,
+		Error,
+		Critical,
+		Alert,
+		Emergency,
+	}
+
+	for i, s := range severities {
```

Comment:
could we just iterate through the severityMap here (instead of possibly getting out of sync in our slice -- if the intent is to check all values)?